src/shared/components/ContextMenu/ContextMenu.tsx

```diff
@@ -27,13 +19,12 @@ export const ContextMenuItem: React.FC<MenuItemProps> = ({ icon, children, onCli
   )
 }
 
-export const ContextMenu: React.FC<ContextMenuProps> = ({ contextMenuOpts: { isActive, position }, children }) => {
-  const { contextMenuContainerRef } = useOverlayManager()
+export const ContextMenu: React.FC<PopoverBaseProps> = ({ children, isVisible, ...rest }) => {
   return (
-    <Portal containerRef={contextMenuContainerRef}>
-      <CSSTransition in={isActive} timeout={150} classNames="menu" unmountOnExit>
-        <StyledContainer position={position}>{children}</StyledContainer>
+    <PopoverBase isVisible={isVisible} {...rest}>
+      <CSSTransition in={isVisible} timeout={150} classNames="menu" unmountOnExit>
+        <StyledContainer>{children}</StyledContainer>
       </CSSTransition>
-    </Portal>
```

Comment:
Not sure which change causing this, but the context menu is broken now: 
![image](https://user-images.githubusercontent.com/51168865/134481956-e04912af-f566-4a77-aef6-32218e90124f.png)
I think it may be due to the fact that we're not using Portal here anymore.

Reply:
z-index seems to do the job here, you think that would be enough or is the Portal imporant?

Reply:
I think it would be better to leave Portal here, but if it requires a lot of work let's just use `z-index`.

Reply:
yeah, that was my thought as well

Reply:
I'm not sure if we understand each other here 😅 . I wanted to keep this portal. 
let's bring it back 😄  opened https://github.com/eldiegod/atlas/pull/3 WDYT?

Reply:
oh lol, ok it's back now